src/Nethermind/Nethermind.Trie/Pruning/ReadOnlyTrieStore.cs

```diff
@@ -52,6 +54,11 @@ public void Set(in ValueHash256 hash, byte[] rlp)
         {
         }
 
+        public void PersistCache(CancellationToken cancellationToken)
+        {
+            _trieStore.PersistCache(cancellationToken);
```

Comment:
Should `ReadOnlyTrieStore` persist anything? 🤔

Reply:
`ITrieStore` probably should not even have this. There should be a separate `PruningTrieStore` which actually have this method.

Reply:
Maybe we should make `IPersistableTrieStore`, put it only there?

Reply:
Done, put into IPruningTrieStore.